samples/dymaptic.GeoBlazor.Core.Sample.Shared/Shared/NavMenu.razor

```diff
@@ -55,6 +55,11 @@
                 <span class="oi oi-arrow-right" aria-hidden="true"></span> Vector Layer
             </NavLink>
         </div>
+        <div class="nav-item px-3">
+            <NavLink class="nav-link" href="layerLists">
+                <span class="oi oi-layers" aria-hidden="true"></span> Layer Lists
```

Comment:
I had already used `oi-layers` for feature-layers above. See if you can find an appropriate different icon in the openiconic set. It can be relatively random, just should be unique. At some point we may want to replace these with custom icons.

Reply:
list icon inputted